app/assets/stylesheets/work_versions.scss

```diff
@@ -0,0 +1,7 @@
+@import 'bootstrap';
+@import 'bootstrap/variables';
```

Comment:
@rschenk having to re-import these each time seems bad, but I'm guessing that ultimately it doesn't matter since it gets compiled down into one file anyway. Having different files named according to the partial they're styling is nice, but I guess the trade off is these duplicated import statements.

Reply:
Yeah I have no idea if Sass is smart enough to compile these all down. I'll have a look

Reply:
I think it is... at least when we precompile the assets in production mode.